core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1386,6 +1387,14 @@ public Void visitTableExecute(TableExecuteNode node, Void context)
             return processChildren(node, context);
         }
 
+        @Override
+        public Void visitSimpleTableExecuteNode(SimpleTableExecuteNode node, Void context)
+        {
+            addNode(node, "SimpleTableExecute", format("[%s]", node.getExecuteHandle()));
```

Comment:
How does it manifest given TableExecutHandle toString is:
```
public String toString()
    {
        return "Execute[" + catalogName + ":" + connectorHandle + "]";
    }
```

Feels like there will be to many `Execute`s :)

Reply:
let's add a test for EXPLAIN ALTER TABLE EXECUTE

Reply:
I will change TableExecutHandle toString to `"TableExecuteHandle[" + catalogName + ":" + connectorHandle + "]"`. If you have better idea please let me know